QuEST/src/QuEST.c

```diff
@@ -834,6 +834,18 @@ Complex calcInnerProduct(Qureg bra, Qureg ket) {
     return statevec_calcInnerProduct(bra, ket);
 }
 
+qreal calcDensityInnerProduct(Qureg bra, Qureg ket) {
```

Comment:
Somewhat pedantic, but with other change (accepting only density matrices), `bra` and `ket` (implying statevectors) would more meaningfully be called `rho` and `omega` (or something conventional for density matrices)

Reply:
Addressed this in acf7f95214544204939c8267381218a72ec98096.